core/input/input_event.cpp

```diff
@@ -325,6 +325,13 @@ Key InputEventKey::get_physical_keycode() const {
 	return physical_keycode;
 }
 
+void InputEventKey::set_physical_key(bool p_bool) {
+	physical_key = p_bool;
+}
+bool InputEventKey::is_physical_key() const {
+	return physical_key;
+}
```

Comment:
Please keep method definitions separated by a line:
```suggestion
void InputEventKey::set_physical_key(bool p_bool) {
	physical_key = p_bool;
}

bool InputEventKey::is_physical_key() const {
	return physical_key;
}
```